src/components/Message.css

```diff
@@ -38,7 +38,15 @@
 
 .message .message-text > p {
```

Comment:
This would make the message content appear on the next line, after the author nickname. Is there any way to use `:first` selector to make the first `p` inline?